packages/core/src/account/__tests__/simple.test.ts

```diff
@@ -102,29 +102,49 @@ describe("Account Simple Tests", () => {
       ]"
     `);
   });
-});
 
-const givenConnectedProvider = ({
-  owner,
-  chain,
-}: {
-  owner: SmartAccountSigner;
-  chain: Chain;
-}) =>
-  new SmartAccountProvider({
-    rpcProvider: `${chain.rpcUrls.alchemy.http[0]}/${"test"}`,
-    chain,
-  }).connect((provider) => {
+  it("should correctly use the account init code override", async () => {
     const account = new SimpleSmartContractAccount({
-      chain,
-      owner,
-      factoryAddress: getDefaultSimpleAccountFactoryAddress(chain),
-      rpcClient: provider,
+      chain: sepolia,
```

Comment:
does the chain need to be sepolia as opposed to mumbai?

Reply:
doesn't matter what the chain is because we don't make any RPC calls (they're stubbed out or short circuited)

Reply:
in the future, we'll want to actually use `localhost` in test with local rundler + anvil running, but we don't support that well yet